static/app/views/projectDetail/projectScoreCards/projectStabilityScoreCard.spec.tsx

```diff
@@ -0,0 +1,151 @@
+      url: `/organizations/${organization.slug}/sessions/`,
+      body: {
+        groups: [
+          {
+            totals: {
+              [SessionFieldWithOperation.CRASH_FREE_RATE_USERS]: 0.99,
+            },
+          },
+        ],
+      },
+      status: 200,
+    });
+
+    render(
+      <ProjectStabilityScoreCard
+        field={SessionFieldWithOperation.CRASH_FREE_RATE_USERS}
+        hasSessions
+        selection={selection}
+        isProjectStabilized
+        query="test-query"
+      />,
+      {
+        organization,
+      }
+    );
+
+    expect(endpointMock).toHaveBeenNthCalledWith(
+      1,
+      `/organizations/${organization.slug}/sessions/`,
+      expect.objectContaining({
+        query: {
+          environment: [],
+          field: SessionFieldWithOperation.CRASH_FREE_RATE_USERS,
+          project: 1,
+          interval: '1d',
+          statsPeriod: '14d',
+          query: 'test-query',
+        },
+      })
+    );
+
+    expect(screen.getByText('Crash Free Users')).toBeInTheDocument();
+
+    expect(await screen.findByText('99%')).toBeInTheDocument();
+  });
+
+  it('renders crash free sessions', async function () {
+    endpointMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/sessions/`,
+      body: {
+        groups: [
+          {
+            totals: {
+              [SessionFieldWithOperation.CRASH_FREE_RATE_SESSIONS]: 0.99,
+            },
+          },
+        ],
+      },
+      status: 200,
+    });
+
+    render(
+      <ProjectStabilityScoreCard
+        field={SessionFieldWithOperation.CRASH_FREE_RATE_SESSIONS}
+        hasSessions
+        selection={selection}
+        isProjectStabilized
+        query="test-query"
+      />,
+      {
+        organization,
+      }
+    );
+
+    expect(endpointMock).toHaveBeenNthCalledWith(
+      1,
+      `/organizations/${organization.slug}/sessions/`,
+      expect.objectContaining({
+        query: {
+          environment: [],
+          field: SessionFieldWithOperation.CRASH_FREE_RATE_SESSIONS,
+          project: 1,
+          interval: '1d',
+          statsPeriod: '14d',
+          query: 'test-query',
+        },
+      })
+    );
+
+    expect(screen.getByText('Crash Free Sessions')).toBeInTheDocument();
+
+    expect(await screen.findByText('99%')).toBeInTheDocument();
+  });
+
+  it('renders without sessions', async function () {
+    endpointMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/sessions/`,
+      body: {
+        detail: 'test error',
+      },
+      status: 404,
+    });
+
+    render(
+      <ProjectStabilityScoreCard
+        field={SessionFieldWithOperation.CRASH_FREE_RATE_SESSIONS}
+        hasSessions={false}
+        selection={selection}
+        isProjectStabilized
+        query="test-query"
+      />,
+      {
+        organization,
+      }
+    );
+
+    expect(endpointMock).not.toHaveBeenCalled();
```

Comment:
consider putting this after the awat findBy because i'm always concerned it just hasn't been called yet